docs/windows_installation.md

```diff
@@ -43,32 +39,31 @@ These Wheels are also used by CI running on windows, and are therefore tested to
 
 Other versions must be downloaded from the above link.
 
-``` powershell
+ powershell
```

Comment:
not sure what you're trying to do - but please use a proper editor that doesn't introduce random changes.
removing the backticks here changes rendering.

You should also have a look at how it renders online - as shown in the [contributors guide](https://www.freqtrade.io/en/latest/developer/#documentation).

Reply:
I use VS Code. I think the markdownlint plugin did that, when I clicked on fix all possible violations. I fixed that in the last commit.